package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
+    "prepare": "huksy install"
+  },
+  "devDependencies": {
+    "husky": "^8.0.0"
+  },
+  "dependencies": {
+    "lint-staged": "^15.2.10"
+  }
```

Comment:
Do we need this package.json here??

Reply:
yes we need because we can only initialize the .husky folder in the root repo